tests/Mapper/ORM/DoctrineORMMapperTest.php

```diff
@@ -90,12 +90,15 @@ public function testLoadClassMetadata(): void
             'length' => 10,
         ];
 
-        static::assertSame(
-            (array) ($classMetadata->discriminatorColumn instanceof DiscriminatorColumnMapping
-                ? DiscriminatorColumnMapping::fromMappingArray($expectedDefinition)
-                : $expectedDefinition),
-            (array) $classMetadata->discriminatorColumn
-        );
+        $expected = (array) ($classMetadata->discriminatorColumn instanceof DiscriminatorColumnMapping
+            ? DiscriminatorColumnMapping::fromMappingArray($expectedDefinition)
+            : $expectedDefinition);
+        ksort($expected);
+        
+        $actual = (array) $classMetadata->discriminatorColumn;
+        ksort($actual);
+
+        static::assertSame($expected, $actual);
```

Comment:
Can't it be simplify to something like
```
ksort($expectedDefinition);

        $actual = (array) $classMetadata->discriminatorColumn;
        ksort($actual);

        static::assertSame($expectedDefinition, $actual);
```

Without the 
```
$expected = (array) ($classMetadata->discriminatorColumn instanceof DiscriminatorColumnMapping
            ? DiscriminatorColumnMapping::fromMappingArray($expectedDefinition)
            : $expectedDefinition);
```
line ?

Reply:
There are some default property values then in the array 

```
1) Sonata\Doctrine\Tests\Mapper\ORM\DoctrineORMMapperTest::testLoadClassMetadata
Failed asserting that two arrays are identical.
--- Expected
+++ Actual
@@ @@
 Array &0 (
+    'columnDefinition' => null
+    'enumType' => null
     'fieldName' => 'discriminator'
     'length' => 10
     'name' => 'discriminator'
+    'options' => Array &1 ()
     'type' => 'string'
 )

/var/www/sonata-doctrine-extensions/tests/Mapper/ORM/DoctrineORMMapperTest.php:97
```

Reply:
And those keys are not set on ORM2 so it does not pass with both versions then